hardware/nic/mellanox/mt27520.pan

```diff
@@ -6,5 +6,5 @@ structure template hardware/nic/mellanox/mt27520;
 'pxe' = false;
 'boot' = false;
 'media' = 'ethernet';
-'name' = 'Mellanox Technologies MT27520 Family [ConnectX-3 Pro]';
+'name' = 'Mellanox Technologies MT27520 Family (ConnectX-3 Pro)';
```

Comment:
Why change this? What does `lspci` say?

Reply:
i think this is the reason why:
cfg/standard/hardware/nic/mellanox/mt27520.pan:9: Unwanted space in simple expression in square brackets
'name' = 'Mellanox Technologies MT27520 Family [ConnectX-3 Pro]';

But I guess this should be fixed in panlint instead

Reply:
Indeed, that test shouldn't match inside a string. I've opened an issue.